client/state/timezones/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { timezonesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+/**
+ * Collect and store timezones data
+ *
+ * @param  {Object} state - Current state
+ * @param  {Object} action - action payload
+ * @return {Object} updated state
+ */
+export const items = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, {
+		rawOffsets,
+		timezonesByContinent,
+	} ) => ( { rawOffsets, timezonesByContinent } ),
+}, timezonesSchema );
+
+/**
+ * Track the timezones requesting process
```

Comment:
❓ - are we actually concerned with knowing if a timezone request succeeded or failed or is in-transit as part of the visually renderable app state?

the middleware layer has all the power it needs to perform actions like HTTP retries and reporting (it can dispatch a notice for example). could we just as easily not track this value in Redux?